src/util.h

```diff
@@ -1098,8 +1098,10 @@ class COSUtil
     {
 #ifdef _WIN32
```

Comment:
Are the Windows and Android ones okay?  Is the `Q_` a "Qt" thing that we can use for all platforms?  What's the `OT_` thing?

Reply:
Check the enum above

Reply:
Windows and android are ok, but probably worth changing to the Qt style.

Reply:
It seems as if we don't use Q_OS_WIN anywhere. Android is now in the same style as used in other places.